mailpoet/assets/js/src/automation/editor/components/panel/step-name.tsx

```diff
@@ -0,0 +1,52 @@
+import { Dropdown, TextControl } from '@wordpress/components';
+import { edit, Icon } from '@wordpress/icons';
+import { dispatch, useSelect } from '@wordpress/data';
+import { PlainBodyTitle } from './plain-body-title';
+import { TitleActionButton } from './title-action-button';
+import { store } from '../../store';
+import { Step } from '../workflow/types';
+
+type Props = {
+  step: Step;
+};
+export function StepName({ step }: Props): JSX.Element {
+  const { stepType } = useSelect(
+    (select) => ({
+      stepType: select(store).getStepType(step.key),
+    }),
+    [],
+  );
+  return (
+    <Dropdown
```

Comment:
This seems like an edge case, but it's possible to get into a state where you can't click outside of the input to close it.

https://user-images.githubusercontent.com/8656640/184237681-4bacee2d-9438-4e6f-8801-926457bb8bb5.mov

Reply:
Can reproduce. This seems to be a more general issue with the `<DropDown />` component. It happens also in the dropdown where you can change the automation name. Not sure, if we can tackle this in this PR :thinking:

Reply:
Agreed that would definitely be out of scope